src/modules/hip/hip_tensor_audio_augmentations.hpp

```diff
@@ -0,0 +1,6 @@
+#ifndef HIP_TENSOR_AUDIO_HPP
```

Comment:
MACRO should match file name for this files
https://github.com/sampath1117/rpp/blob/sr/to_decibels_hip/src/modules/hip/hip_tensor_audio_augmentations.hpp

So it should be changed to HIP_TENSOR_AUDIO_AUGMENTATIONS_HPP

Reply:
Done